3rdparty/quirc/src/quirc.cpp

```diff
@@ -0,0 +1,132 @@
+/* quirc -- QR-code recognition library
+ * Copyright (C) 2010-2012 Daniel Beer <dlbeer@gmail.com>
```

Comment:
What is a source of this file? Is that a part of Quirc library or your code?

Am I right that project is dedicated to replacing quirc to OpenCV implementation of QR codes decoding?